### PR TITLE
Schboop

### PR DESCRIPTION
**Pull request by katlande**
Closed my previous PR and setting up a new one from patch-1. Thank you for your help! 

Original description:
This is a fan-made plugin for OSRS youtuber Schboop's community. It plays sound clips from Schboop's videos in reaction to various triggers.

### CODE REVIEW

File: plugins/Schboop [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/katlande/Schboop.git
```
Comment (YvesW) [inline plugins/Schboop:1]:
```suggestion
repository=https://github.com/katlande/kat-RL-plugins.git
```

**Comment by YvesW** [comment]
img

**Comment by katlande** [comment]
Not sure how to fix this:

schboop: icon.png is too high-resolution. It should be 48x72 px
in file /tmp/pluginhub-package16890144763410240947/schboop/repo/icon.png

I updated the original image in the Schboop branch, but I don't know where this /tmp folder is or how I can modify

I appreciate your help so much sorry!!

**Comment by iProdigy** [comment]
You need to update the commit hash for the CI to see the updated version with the smaller image

**Comment by YvesW** [comment]
You forgot to update the hash in this plugin file indeed.

Additionally, I think you included the pluginhub files and folders in your plugin branch. It should only contain the plugin files; you can refer to the example-plugin here: https://github.com/runelite/example-plugin

Btw I kinda griefed you because the images in your readme don't work anymore due to the repo name change.

**Comment by katlande** [comment]
> Not an in-depth review, but can you tell us what licenses are applicable to these sound files, and are you adhering to them? If it's clips of a streamer's voice, we'd like to see proof that the streamer allows the use of the clips.

They're all sounds from his videos. He is aware and on board with the creation of this plugin. Here's a clip from his most recent stream where he talks about how excited he is for the plugin: https://youtu.be/HFISRaXnT2c?t=2319

I can also message him on discord if you want him to directly approve the use of his clips.

**Comment by katlande** [comment]
Many of the functions of this plugin were brainstormed in this comment thread on one of his videos. He liked all the ideas and comments on the thread:
comments


I messaged him on discord too, but sometimes he takes a few days to respond.

**Comment by katlande** [comment]
Okay here's his direct approval:

gottem

Let me know if you guys need anything else on this

**Comment by LlemonDuck** [comment]
Your license is attributing everything to Adam, which isn't really an issue but do you want to change that before we merge in?

**Comment by katlande** [comment]
Good catch, updated now thanks!!

**Pull request merged**